src/app/workloads/workloads.component.ts

```diff
@@ -443,7 +438,6 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
     this.captchaSuccess = true;
     this.captchaIsExpired = false;
     console.log(this.captchaSuccess);
```

Comment:
Why we are putting console here?

Reply:
for test purpose  , now i remove this !!